pre/CMakeLists.txt

```diff
@@ -2,9 +2,16 @@ add_subdirectory(NSIDC-OSTIA_SST-ICE_blend)
 add_subdirectory(prepare_ocnExtData)
 
 file(GLOB pythonscripts CONFIGURE_DEPENDS ./remap_restart/remap*  ./remap_restart/bin2nc_merra2* ./remap_restart/tests/*.*)
-install(
-   PROGRAMS ${pythonscripts}
-   DESTINATION bin)
+
+foreach (file ${pythonscripts})
+  if ( NOT ${file} MATCHES "./remap_restart/remap_utils.py") 
+    install(PROGRAMS ${file} DESTINATION bin)
+  endif()
+endforeach()
+
```

Comment:
This is an unusual construct.   If that one file should not be installed, perhaps it should be moved to a different directory or deleted.  Why is it special?

Reply:
I think it's special because it now uses `configure_file` to alter it and then you install the `${CMAKE_CURRENT_BINARY_DIR}/${file}` version.

Probably the right thing to do is not use a glob and explicitly list the files that can be installed the "usual" way. But that does get tedious first time out.